src/main/java/com/microsoft/aad/msal4j/AuthorizationRequestUrlParameters.java

```diff
@@ -187,11 +199,20 @@ public Builder scopes(Set<String> val){
          * policy has not been met,{@link MsalServiceException} will contain claims that need be
          * consented to.
          */
-        public Builder claims(Set<String> val){
+        public Builder claims(String val){
             this.claims = val;
             return self();
         }
 
+        /**
+         * Allows the client to indicate to the token service what policies or features it is
+         * capable of handling
+         */
+        public Builder clientCapabilities(Set<String> val){
```

Comment:
Might have missed the latest discussion around this, but I thought client capabilities should be added to the client application object?